frontend/src/components/SearchResult.js

```diff
@@ -5,22 +5,28 @@ import { IoIosAlarm } from 'react-icons/io';
 import { IoBarbellSharp } from 'react-icons/io5';
 import { useEffect, useState } from 'react';
 import { getRecipesWithMatchingIngredients } from './apiCalls';
+import ModalForSearch from './ModalForSearch';
 
 function SearchResult({ ingredients, searchBy }) {
-  const [recipe, setRecipe] = useState([]);
+  const [recipe, setRecipe] = useState(null);
+  const [show, setShow] = useState(false);
+  const handleShow = () => setShow(true);
 
   useEffect(() => {
     const getRecipes = async () => {
       const recipeList = await getRecipesWithMatchingIngredients(
         ingredients,
         searchBy
       );
-      recipeList
-        ? recipeList.sort((a, b) => a.name.localeCompare(b.name))
-        : console.log('Loading');
+      recipeList?.sort((a, b) => a.name.localeCompare(b.name));
+
       setRecipe(recipeList);
```

Comment:
Ha már recipeList-et seteltek, akkor lehetne a state neve is recipeList vagy recipes recipe helyett.

Reply:
javítva recipes - re